grpc-testtool/README.md

```diff
@@ -19,3 +19,23 @@ There are 3 RPC specs that must be implemented:
 2. The sync proto, which supports retrieving range and change proofs
 3. The process-server proto, which currently only retrieves metrics
 
+# Running
+
+To run a single test and make sure things are working, first check out and build the go and rust code.
+See the corresponding README for specific build requirements.
+
+ - BASE=$HOME
+ - cd $BASE && git clone git@github.com:ava-labs/merkledb-tester.git
+ - cd $BASE && git clone git@github.com:ava-labs/firewood.git
```

Comment:
````suggestion
```sh
BASE=<your project directory>
cd $BASE && git clone git@github.com:ava-labs/merkledb-tester.git
cd $BASE && git clone git@github.com:ava-labs/firewood.git
```
````

However, I think showing someone how to clone repos is overkill. You're going to throw someone for a loop if they haven't set up ssh, for example. I think it would be better to simply provide a link to the `merkledb-tester` repo and mention that they must be contained in the same directory (if that is in fact a requirement).

Reply:
Good suggestion for formatting. I want some copy/paste examples. Although it's not a hard requirement for having them in the same base directory, it would be harder to make copy/paste code that shows that.